### PR TITLE
feat(samples): add sample code for ListAssets v1p5beta1

### LINKED ISSUE #354: Add sample code for ListAssets v1p5beta1
Add sample code for ListAssets v1p5beta1.

### PR DESCRIPTION
**Pull request by Larittic-GG**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-asset/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #354 🦕

### CODE REVIEW

File: samples/listAssets.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  // [START asset_quickstart_list_assets]
+  const util = require('util');
+  const {v1p5beta1} = require('@google-cloud/asset');
+  const client = new v1p5beta1.AssetServiceClient();
+
+  const projectId = await client.getProjectId();
+  const projectResource = `projects/${projectId}`;
+  // TODO(developer): Choose types of assets to list, such as 'storage.googleapis.com/Bucket':
+  //   const assetTypesList = ['storage.googleapis.com/Bucket', 'bigquery.googleapis.com/Table'];
+  // Or simply use empty list to list all types of assets:
```
Comment (bcoe) [inline samples/listAssets.js:33]: what we tend to do in comments here, is leave the expected variable commented out, such that someone could simply remove the `//` and have a functional sample:

https://github.com/googleapis/nodejs-secret-manager/blob/master/samples/getSecretVersion.js#L22

File: samples/listAssets.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      contentType: 'RESOURCE',
+      // (Optional) Add readTime parameter to list assets at the given time instead of current time:
+      //   readTime: { seconds: 1593988758 },
+    };
+
+    // Call cloud.assets.v1p5beta1.ListAssets API.
+    const result = await client.listAssets(request);
+    // Handle the response.
+    console.log(util.inspect(result, {depth: null}));
+    // [END asset_quickstart_list_assets]
```
Comment (bcoe) [inline samples/listAssets.js:50]: I believe this should be moved down to below `listAssets()`;, such. that the code between `START` and `END` is a self contained functional block.

**Review by bcoe — CHANGES_REQUESTED** [review]
Noticed a couple things, let me know your thoughts.

File: samples/listAssets.js (hunk 2 above)
Comment (Larittic-GG) [reply samples/listAssets.js]: Thanks for catching this! I was following the samples for other APIs. Now it seems the other samples need updating too.

File: samples/listAssets.js (hunk 1 above)
Comment (Larittic-GG) [reply samples/listAssets.js]: Got it. I changed the comments to use "const assetTypes = ..." directly, in order to align with the params used below.

**Review by bcoe — APPROVED** [review]

**Pull request merged**